### ISSUE TITLE
Scan threshold

### ISSUE THREAD

**Issue by yakiler** [eslint/eslint#13928]
**Please describe what the rule should do:**
When the rule is applied to an earlier project, there may be many warnings reported. Should there be a configuration item that will suspend scanning when a certain number of warnings are scanned, or the console log controls the output quantity limit

**What new ECMAScript feature does this rule relate to?**

**What category of rule is this? (place an "X" next to just one item)**

[ ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**

```js

```

**Why should this rule be included in ESLint (instead of a plugin)?**

**Are you willing to submit a pull request to implement this rule?**

**Comment by aladdin-add** [comment]
hi, is the [`--max-warnings`](https://eslint.org/docs/user-guide/command-line-interface#-max-warnings) that you are asking?

**Comment by nzakas** [comment]
There isn't enough information here to really comment on the proposal, so closing.
